sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -407,13 +463,22 @@ class PartialDependenceDisplay:
 
     """
     def __init__(self, pd_results, features, feature_names, target_idx,
-                 pdp_lim, deciles):
+                 pdp_lim, deciles, individual=False, subsample=None):
```

Comment:
I think we should restrict the default to 1000 (or 10000?).

```suggestion
                 pdp_lim, deciles, individual=False, subsample=1000):
```

I am pretty sure that matplotlib would have trouble displaying more than a few thousands of lines and the plot would not be very readable anyway.

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/d8e71120198879e8fad701c4c382d86dc2e22a25